### ISSUE TITLE
Clean up test data sets

### ISSUE THREAD

**Issue by ChristianZimpelmann** [iza-institute-of-labor-economics/gettsim#336]
### Current and desired situation

Currently, the test data sets contain columns that occur neither in the DAG nore as input columns.

### Proposed implementation

- There should only be columns present, which actually occur in the DAG / input columns. 
- If possible even make a test for the test data.
- See #282, for a proposed larger change.

**Comment by hmgaudecker** [comment]
Sounds good. Fine to just convert to yaml in #282, Should all be pretty easy to do programmatically, not much manual input required. I would think:

1. Read test data from csv/ods in pandas, save with same name but as yaml
2. Write test for column names that automatically loops over all test data (skipping any intentionally malformed datasets)
3. Adjust yaml files, either by (read into pandas, delete unnecessary columns, saving under same name) or by hand if yaml structure allows for it.

Should be easy for an RA, maybe with the exception of 2.?

**Comment by ChristianZimpelmann** [comment]
Agreed! Let's work on this after #275 is merged and v0.5 is released.
